packages/composables/src/useCartItem.ts

```diff
@@ -76,19 +115,20 @@ export function useCartItem(cartItem: Ref<LineItem>): UseCartItemReturn {
   const isPromotion = computed(() => cartItem.value.type === "promotion");
 
   async function removeItem() {
-    const result = await removeCartItem(cartItem.value.id, apiInstance);
-    // broadcastUpcomingErrors(result);
-    await refreshCart();
+    await removeCartItem(cartItem.value.id, apiInstance);
+    refreshCart();
```

Comment:
cart operation is returning cart, what would you say to do something like this?
```suggestion
    const newCart = await removeCartItem(cartItem.value.id, apiInstance);
    await refreshCart(newCart);
```

and then in useCart
```ts
async function refreshCart(newCart?: Cart): Promise<Cart> {
    if (newCart) {
      _storeCart.value = newCart;
      return newCart;
    }
    const result = await getCart(apiInstance);
    _storeCart.value = result;
    return result;
  }
```
so we can skip additional server request?

Reply:
good improvement. I will do 🚀